compiler/noirc_driver/src/lib.rs

```diff
@@ -66,10 +66,16 @@ pub struct CompileOptions {
     #[arg(long = "force")]
     pub force_compile: bool,
 
-    /// Emit debug information for the intermediate SSA IR
+    /// Emit debug information for the intermediate SSA IR to stdout
     #[arg(long, hide = true)]
     pub show_ssa: bool,
 
+    /// Emit the unoptimized SSA IR to file.
+    /// The IR will be dumped into the workspace target directory,
+    /// under `[compiled-package].ssa.json`.
+    #[arg(long, hide = true)]
```

Comment:
```suggestion
    #[arg(long)]
```
Why hide it? I feel like we hide too many flags.

Reply:
Just cause this is a pretty niche flag that I don't think applies to 99% of Noir consumers. However, I don't have to hide it if we want.